src/useCookieValue/__docs__/story.mdx

```diff
@@ -1,30 +1,30 @@
-import { Canvas, Meta, Story, Source } from '@storybook/addon-docs';
-import { Example } from './example.stories';
+import {Canvas, Meta, Source, Story} from '@storybook/addon-docs';
```

Comment:
Interesting that the space between brackets and imports were removed here but not in other files. Any ideas why?

Reply:
They get removed in almost all .mdx files I edit. I think the formatting rules have changed since these files have been last edited.